File: Java/FuncionarioHorista.java
public class FuncionarioHorista extends Funcionario{

    private double horas;

    public FuncionarioHorista(String nome,double salario,double horas){
        super(nome,salario);
        this.horas = horas;
    }

    @Override
    public void calcularSalario() {
        double salarioBase = getSalario();

        double salarioFinal = salarioBase * horas;
        System.out.println("Salário do funcionário " + getNome() + " R$: " + salarioFinal);
    }
}

File: Python/FuncionarioHorista.py
from Funcionario import Funcionario


class FuncionarioHorista(Funcionario):
    def __init__(self, nome, salario, horas):
        super().__init__(nome, salario)
        self.horas = horas

    def calcular_salario(self):
        salario_final = self.salario * self.horas
        print(f"Salário do funcionário {self.get_nome()} é R$: {salario_final}")

